nemoguardrails/library/autoguard/actions.py

```diff
@@ -34,14 +36,12 @@
     "racial_bias_detection": "Racial bias",
     "jailbreak_detection": "Jailbreak attempt",
     "intellectual_property": "Intellectual property",
-    "factcheck": "Factcheck violation",
 }
 
 DEFAULT_CONFIG = {
     "pii_fast": {
         "mode": "OFF",
         "mask": False,
-        "coreference": False,
```

Comment:
intentional?

Reply:
In the current version, we are not providing the users to enable mask or coreference
@rahm-decypher said that it will be done in a future commit

Reply:
both mask and coreference, thats why I am setting both of them to False, although we can discuss about the default config a bit more